### PR TITLE
1157: Add promote ios

### PR DESCRIPTION
**Pull request by ztefanie**
### Short description

Added ios promote to CD

### Proposed changes

- Added ios promote job
- Added ios promote job to native promotion workflow
- added fastlane lane for promoting
- cleaned up notify command

### Side effects

None, not yet tested, will test, when Martin approved the betas.

### Resolved issues

Fixes: #1157

### CODE REVIEW

**Comment by ztefanie** [comment]
tested here: https://app.circleci.com/pipelines/github/digitalfabrik/entitlementcard/5134/workflows/36191f68-696e-457d-a0ba-fbc6e80594a4
works as expected.

**Comment by f1sh1918** [comment]
@ztefanie i merged a fix for the promote administration_backend in this branch!

**Comment by ztefanie** [comment]
> @ztefanie i merged a fix for the promote administration_backend in this branch!

Why?

File: .circleci/src/jobs/promote_ios.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Promotes the app from Testflight to the Apple App Store.
+parameters:
+  build_config_name:
+    type: enum
+    enum: [bayern, nuernberg]
+    default: bayern
+macos:
+  xcode: 15.0.1
```
Comment (f1sh1918) [inline .circleci/src/jobs/promote_ios.yml:8]: we should use xcode 15.2.0 as we do for all other jobs

**Review by f1sh1918 — COMMENTED** [review]
please adjust the xcode version. rest looks fine

File: .circleci/src/jobs/promote_ios.yml (hunk 1 above)
Comment (ztefanie) [reply .circleci/src/jobs/promote_ios.yml:8]: PR is older than this updates, but will adjust it.

**Pull request merged**